hawkular-wildfly-agent-feature-pack/src/main/resources/modules/system/layers/base/org/hawkular/agent/main/module.xml

```diff
@@ -83,6 +83,7 @@
     <module name="org.jboss.vfs"/>
 
     <!-- other dependencies we need -->
+    <module name="javaee.api"/> <!-- bus-commons -->
```

Comment:
> Let me have a look if this could be done differently.

Given my previous statement https://github.com/hawkular/hawkular-agent/pull/137/files#diff-05665f6a96889fd9eb46439a463b6646R26 as long as this feature pack depends on wildfly-feature-pack, the  `javaee.api` module is guaranteed to be there and I do not need to look for another solution.